native/src/binaryRead.h

```diff
@@ -792,6 +792,10 @@ struct RouteDataObject {
 		}
 		return false;
 	}
+
+	std::string toString() {
```

Comment:
getOsmId

Reply:
it is based on RouteDataObject toString - simple string just for debug and logs